src/main/java/Main.java

```diff
@@ -0,0 +1,50 @@
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.*;
+import org.hibernate.query.Query;
+import org.hibernate.cfg.Configuration;
+
+import javax.persistence.metamodel.EntityType;
+
+@Slf4j
+public class Main {
```

Comment:
Choose a more descriptive name than Main.  Main is considered bad practice for a class name.
* Easy to confuse with the main method.
* Not descriptive of what it does.
* Ambiguous; there are likely many classes named Main.  This will cause merge conflicts.